src/platform/utilities/api/index.js

```diff
@@ -18,6 +33,7 @@ export function fetchAndUpdateSessionExpiration(...args) {
         if (sessionExpiration) {
           localStorage.setItem('sessionExpiration', sessionExpiration);
         }
+        refreshSSOeSession();
```

Comment:
this could go elsewhere, but it seemed to make sense to group it with the current `vets-api` logic